src/main/java/ch/njol/skript/classes/data/BukkitEventValues.java

```diff
@@ -1530,5 +1531,25 @@ public Location get(LootGenerateEvent event) {
 				}
 			}, EventValues.TIME_NOW);
 		}
+
+		// PlayerShearEntityEvent - 1.15+ event values
```

Comment:
you could be checking with `#ClassExists` method vs. `#getMinor()` since in your current one you also have the issue of `2.0` causing it to break if ever released

Might be wrong on `classExists` might actually be `MethodExists` since the event was 1.13 while methods used are 1.15